x-pack/plugins/secops/public/components/loading/index.tsx

```diff
@@ -17,37 +17,50 @@ injectGlobal`
   }
 `;
 
+type Position = 'relative' | 'absolute';
+
+type ZIndex = number | '-moz-initial' | 'inherit' | 'initial' | 'revert' | 'unset' | 'auto';
```

Comment:
This should be a props below in and this removed: `style={{`, and then this line can disappear.